xds/internal/xdsclient/xdsresource/type_cds.go

```diff
@@ -56,6 +58,72 @@ type ClusterLBPolicyRingHash struct {
 	MaximumRingSize uint64
 }
 
+// OutlierDetection is the OutlierDetection configuration for a cluster.
```

Comment:
`// OutlierDetection is the outlier detection configuration for a cluster` instead?

Reply:
Switched.